workflow/artifacts/logging/driver.go

```diff
@@ -30,6 +31,10 @@ func (d driver) Load(a *wfv1.Artifact, path string) error {
 	return err
 }
 
+func (d driver) OpenStream(inputArtifact *wfv1.Artifact) (io.ReadCloser, error) {
+	return common.LoadToStream(inputArtifact, d)
```

Comment:
This is wrong, the driver is a decorator, and should delegate all calls to `d.delegate`. It should also logs the same kind of info as `Load`.